src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java

```diff
@@ -651,6 +627,13 @@ public static void buildMapTableView(TableView<MapFX> tableView, ObservableList<
         tableView.getColumns().add(authorColumn);
         extractors.put(authorColumn, mapFX -> mapFX.getAuthor().getLogin());
 
+        TableColumn<MapFX, Boolean> recommendColumn = new TableColumn<>("Recommended");
+        recommendColumn.setCellValueFactory(o -> o.getValue().recommendedProperty());
+        recommendColumn.setCellFactory(CheckBoxTableCell.forTableColumn(recommendColumn));
+        recommendColumn.setMinWidth(100);
+        tableView.getColumns().add(recommendColumn);
+        extractors.put(recommendColumn, MapFX::isRecommended);
```

Comment:
I don't think we want to offer copying boolean values via context menu. Please remove the extractors.put